### ISSUE TITLE
Single datepicker hides cancel/apply buttons even when autoApply is false

### ISSUE THREAD

**Issue by andyfurniss4** [dangrossman/daterangepicker#1872]
I want to have a single datepicker with the apply/cancel functionality but it seems that regardless of whether I set autoApply to false, the buttons are always hidden when singleDatePicker is true.

**Comment by Solybum** [comment]
For some reason, if singleDatePicker is enabled but timePicker is not, it will add auto-apply as seen here
https://github.com/dangrossman/daterangepicker/blob/bc645d93e837905dcd38b04b8d9123978ea9f571/daterangepicker.js#L384-L392

**Comment by dangrossman** [comment]
Because these buttons are not needed in that situation. There is no chance of a partial selection.

**Comment by Solybum** [comment]
I was looking about having the input to be cleared, which if I understood correctly, requires 'autoUpdateInput': false, and 'autoApply': false, so the user can remove the content from the input and not have it be filled again automatically.

**Comment by Solybum** [comment]
Actually, nvm, it seems you can instead just populate the field from the callback, and letting the user delete the input on their own, although for my use case I might want to use the cancel button as a clear instead.

**Comment by jonathantribouharet** [comment]
Any news on this?
I need to have a simple way for users to clear the input, the clear button seems perfect.
Here my code:

```
    datepicker.daterangepicker({
      singleDatePicker: true,
      autoApply: false,
      autoUpdateInput: false,
      locale: {
        format: 'DD/MM/YYYY',
        cancelLabel: 'Clear'
      }
    });
```

**Comment by dangrossman** [comment]
I don't expect to change this. I would suggest either changing it in your copy, or putting your clear button on your form, instead of on the date picker dropdown widget.

**Comment by TBSliver** [comment]
I have added a PR which fixes this issue, #1931 (Ignore the earlier one, Github has decided to default to allow PR's on forks instead of back to the main repo and I didnt notice until to late).

If this change is not wanted, then the documentation should be updated to reflect that the singleDatePicker cannot be used with this option, to save confusion.
